contracts/Uniswap/UniswapAnchoredView.sol

```diff
@@ -158,34 +164,34 @@ contract UniswapAnchoredView is UniswapConfig {
     }
 
     /**
-     * @dev Fetches the token/eth price accumulator for a config, with same decimals as the token.
+     * @dev Fetches the current token/eth price accumulator from uniswap, with same decimals as the token.
```

Comment:
the decimals for this comment has changed, now it has w/e decimals uniswap normally gives... I guess it looks like it uses decimals from right side of the pair?

Reply:
Ah yes just saw that in spec but forgot to update the comment here